ext/standard/crc32.c

```diff
@@ -69,24 +69,13 @@ static uint32_t crc32_aarch64(uint32_t crc, char *p, size_t nr) {
 # pragma GCC pop_options
 #endif
 
-/* {{{ Calculate the crc32 polynomial of a string */
-PHP_FUNCTION(crc32)
+uint32_t crc32_bulk_update(uint32_t crc, const char *p, size_t nr)
 {
-	char *p;
-	size_t nr;
-	uint32_t crcinit = 0;
-	uint32_t crc;
-
-	ZEND_PARSE_PARAMETERS_START(1, 1)
-		Z_PARAM_STRING(p, nr)
-	ZEND_PARSE_PARAMETERS_END();
-
-	crc = crcinit^0xFFFFFFFF;
-
 #if HAVE_AARCH64_CRC32
 	if (has_crc32_insn()) {
 		crc = crc32_aarch64(crc, p, nr);
-		RETURN_LONG(crc^0xFFFFFFFF);
+		nr = 0;
+		p += nr;
```

Comment:
At this point, why not just return `crc`? 

Thanks

Reply:
done